src/components/StatementCheckboxes/index.js

```diff
@@ -0,0 +1,41 @@
+import {useFormikContext} from 'formik';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import StatementCheckbox from 'components/StatementCheckboxes/StatementCheckbox';
```

Comment:
```suggestion
import StatementCheckbox from './StatementCheckbox';
```
I'd say a relative import is a bit more appropriate since it's all grouped around the same main public component?